cts/cts-log-watcher.in

```diff
@@ -43,9 +43,6 @@ if __name__ == '__main__':
             skipthis=1
             prefix = args[i+1]
 
-        elif args[i] == '-t' or args[i] == '--tag':
-            skipthis=1
-
     if not os.access(filename, os.R_OK):
         print(prefix + 'Last read: %d, limit=%d, count=%d - unreadable' % (0, limit, 0))
         sys.exit(1)
```

Comment:
It's nothing that needs to happen in this PR, but cts-log-watcher.in sure does need to be updated at least stylistically, but also using argparse would be great.

Reply:
definitely